plugins/CoreHome/stylesheets/dataTable/_dataTable.less

```diff
@@ -153,10 +153,13 @@ table.subActionsDataTable td.label,
 table.actionsDataTable td.label {
   border-top: 0;
   border-left: 0;
-  position: sticky; position: -webkit-sticky;
   left: 0;
   z-index: 1;
   background-clip: padding-box;
+
+  &:first-child {
```

Comment:
@katebutler I reckon this overwrites https://github.com/matomo-org/matomo/pull/15129/files#diff-8b20fa1b843d28bf8988e7c3c93a54b2R661 and breaks the other feature? 

Could we maybe simply add a new class to those column headers that does a textalign:left  ? This should reduce the risk of breaking things